docs/audits.md

```diff
@@ -2,7 +2,7 @@
 
 <h1 align="center">Boson Protocol V2</h1>
 
-### [Intro](../README.md) | Audits | [Setup](setup.md) | [Tasks](tasks.md) | [Architecture](architecture.md) | [Domain Model](domain.md) | [State Machines](state-machines.md) | [Happy Path Exchange](happy-path-exchange.md)
+### [Intro](../README.md) | Audits | [Setup](setup.md) | [Tasks](tasks.md) | [Architecture](architecture.md) | [Domain Model](domain.md) | [State Machines](state-machines.md) | [Sequences](sequences.md)
 
 ## Audits
 Version 2 of the Boson Protocol is an entirely new codebase and architecture from V1 and so we have once again had the  protocol thoroughly audited from top to bottom. Here are the results of those audits.
```

Comment:
We still have "coming soon" for Omniscia. We need to put a link to the actual report.